test/lit/exec/read-nn-null.wast

```diff
@@ -30,6 +30,12 @@
 (module
   (import "fuzzing-support" "log-i32" (func $log (param i32)))
 
+  ;; NORMAL:      [fuzz-exec] calling foo
+  ;; NORMAL-NEXT: [LoggingExternalInterface logging 42]
+  ;; NORMAL-NEXT: [trap unreachable]
+  ;; TNH:      [fuzz-exec] calling foo
+  ;; TNH-NEXT: [LoggingExternalInterface logging 42]
+  ;; TNH-NEXT: [trap unreachable]
```

Comment:
These lines seem to duplicate new lines 64-67? Might be a pre-existing problem before this PR, but it's odd this PR just hoists one of them and not the other.

Reply:
Do you know why the execution log is duplicated in the first place? I think only the first repetition was hoisted because the update script only knows to pair a single ouput item with a given input item.

Reply:
No idea. But from what you say the updater script is doing the right thing, so it's definitely a separate issue.